misc/python/materialize/checks/drop_subsource.py

```diff
@@ -0,0 +1,89 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+from textwrap import dedent
+from typing import List
+
+from materialize.checks.actions import Testdrive
+from materialize.checks.checks import Check
+from materialize.checks.common import KAFKA_SCHEMA_WITH_SINGLE_STRING_FIELD
+
+
+class DropSubsource(Check):
+    def initialize(self) -> Testdrive:
+        return Testdrive(
+            dedent(
+                """
+                > CREATE SECRET pgpass AS 'postgres'
```

Comment:
This is likely going to conflict with the PgCdc Check. You need a dedicated secret/connection/etc, everything.

Reply:
Adjusted some, migtht need to address further objects.